### ISSUE TITLE
Question about CGA Implementation in the Code

### ISSUE THREAD

**Issue by jupitercsd** [Fir-lat/LaVer#6]
### Reminder

- [x] I have read the above rules and searched the existing issues.

### System Info

Hi, thanks for releasing the LaVer code! I noticed a discrepancy between the paper and the code that I'd like to clarify. The paper introduces Clipped Gram-Anchoring (CGA) in Eq. 8: $$L_{CGA}=||CLIP(G(Z)-G(\widehat{Z}))||_F^2$$, However, in src/llamafactory/model/laver/modeling_laver.py (lines 354–360), the Gram loss is computed as: `loss_gram = F.mse_loss(matrix_student, matrix_teacher)` , This corresponds to GA (Eq. 7) rather than CGA (Eq. 8) — the Clip operation is missing. I searched the whole repo but couldn't find the clipping implementation anywhere.

### Reproduction

```text
Put your message here.
```

**Comment by Fir-lat** [comment]
Thanks for catching this! The previous code computes GA and is missing the element-wise clipping described in Eq. (8)

The CGA can be simply implemented by replacing with this:
```python3
loss_gram = F.relu(matrix_student - matrix_teacher).square().mean()
```

Here, .mean() preserves the normalization used by the existing F.mse_loss; the squared Frobenius norm in Eq. (8) uses .sum(), differing by a scaling factor. You can try both variants and find the best practice for your schemes!

This is now fixed: https://github.com/Fir-lat/LaVer/blob/main/src/llamafactory/model/laver/modeling_laver.py#L364
